CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Changelog
+## 2.1.2 (2017-06-25)
+
+- Github ssh keys
```

Comment:
Also please feel free to add your github login (@andreimc) so that proper contribution credit can be given.